tests/e2e/specs/dashboard-samples/Documentation.spec.ts

```diff
@@ -88,7 +88,7 @@ suite(`Check links to documentation page in Dashboard ${BASE_TEST_CONSTANTS.TEST
 
 			Logger.trace(`Current URL: ${currentUrl}`);
 
-			expect(link.href, `${link.href} not includes ${currentUrl}`).oneOf([currentUrl, currentUrl + '/']);
+			expect(link.href, `${link.href} not includes ${currentUrl}`).oneOf([currentUrl, currentUrl + '/', currentUrl.replace("/en/documentation/","/documentation/en-us/") + '/']);
```

Comment:
It seems to be a bug in the Dashboard, which should be documented as a JIRA ticket.
Our test could fail because of that.

Reply:
done.